src/software/field_tests/movement_field_test.py

```diff
@@ -0,0 +1,119 @@
+import pytest
+
+import software.python_bindings as tbots
+from proto.ssl_gc_common_pb2 import Team
+from proto.import_all_protos import *
+from software.field_tests.field_test_fixture import *
+from software.simulated_tests.tbots_test_fixture import *
+from software.simulated_tests.pytest_main import pytest_main
+from software.logger.logger import createLogger
+from software.simulated_tests.robot_enters_region import RobotEventuallyEntersRegion
+from proto.message_translation.tbots_protobuf import create_world_state
+
+logger = createLogger(__name__)
+
+
+# this test can be run either in simulation or on the field
+# @pytest.mark.parametrize(
+#     "robot_x_destination, robot_y_destination",
+#     [(-2.0, -1), (-2.0, 1.0), (0.0, 1.0), (0.0, -1.0)],
+# )
+@enable_thunderscope
+def test_basic_movement(simulated_test_runner):
+
+    robot_starting_x = 0
+    robot_starting_y = 0
+    ROBOT_ID = 0
```

Comment:
looks like we're hard coding this to robot id is 0, hopefully in the future we can update the code to automatically pick up a robot on the network